pt-PT.json

```diff
@@ -308,7 +308,7 @@
 	"INTRO_TASTE_MUSIC": "Música",
 	"INTRO_TASTE_TRAVEL": "Viagens",
 	"INTRO_TASTE_LATE_NIGHT": "Late Night Shows",
-	"INTRO_TASTE_REALITY": "Realidade",
+	"INTRO_TASTE_REALITY": "Reality Show",
```

Comment:
I see that this text was translated back to english, is that correct?

Reply:
Yes, because what was was a literal translation, but that doesn't make sense and in Portugal we always use "Reality Show" to designate this type of program.